### ISSUE TITLE
need a method to measure goodness-of-fit of the model

### ISSUE THREAD

**Issue by rainchen** [daugaard/linear-regression#2]
I think R-squared is a commonly used metric, can you implement this method?
Here is a reference: http://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html
